src/tests/integration/resolvers/customConsul.ts

```diff
@@ -0,0 +1,379 @@
+import { Catalog, KvStore } from '@creditkarma/consul-client'
```

Comment:
Thanks for the change. I'm wondering if this test is overkill for this feature. I was thinking more that you could create mock resolvers that just took in some hard-coded data instead of full-featured consul/vault resolvers. Though it's better to be overkill than not enough...